packages/ia-pic-uploader/src/ia-pic-uploader.ts

```diff
@@ -686,8 +690,8 @@ export class IAPicUploader extends LitElement {
       }
 
       .profile-section > .full-preview img {
-        max-height: 100px;
-        max-width: 200px;
+        max-height: ${imgMaxHeight};
+        max-width: ${imgMaxwidth};
```

Comment:
Yes i know the idea of using `object-fit: contains`. since we are using pic-uploader for different pages and having different styling, I had to pass max-height/max-width property specifically for collection pages to make it work.